Swiftfin/Components/PosterButton.swift

```diff
@@ -63,6 +63,7 @@ struct PosterButton<Item: Poster>: View {
                 .backport
                 .matchedTransitionSource(id: "item", in: namespace)
                 .posterShadow()
+                .accessibilityElement(children: .ignore)
```

Comment:
We're gonna have to think more about how to properly set `PosterButton`, as there are a variety of states we have to consider where this direct ignore may not be correct.

- when `Show poster labels` is disabled, the title text is hidden from the label. Instead, an abbreviated title is shown on the title in the placeholder/failure state.
- overlay elements like watched count, progress, or favorited